articles/multifactor-authentication/api/manage.md

```diff
@@ -22,7 +40,7 @@ To get a list of the authenticators you've associated and can be used with your
 	"url": "https://${account.namespace}/mfa/authenticators",
 	"headers": [{
 		"name": "Authorization",
-		"value": "Bearer YOUR_API_ACCESS_TOKEN"
+		"value": "Bearer MFA_API_ACCESS_TOKEN"
```

Comment:
This looks like a good old-fashioned, access token and I would strongly recommend naming this what it is: YOUR_ACCESS_TOKEN

We use terms like "Management APIv2 Token" and "MFA API Token" and we make people think that this is some other type of token, while it's **not**. Same old access token, only the scopes change. Follow that please also in the Authentication API PR.

Ignore my comment if this is not an access token but sth else.